packages-native/progress-bar/src/ProgressBar.tsx

```diff
@@ -26,7 +27,9 @@ export class ProgressBar extends Component<Props> {
                     style={this.styles.bar}
                 />
                 {validationMessages.length > 0 && (
-                    <Text style={this.styles.validationMessage}>{validationMessages.join("\n")}</Text>
+                    <Text testID={`${this.props.name}$caption`} style={this.styles.validationMessage}>
```

Comment:
This text component is for validation messages, so I think you shouldn't refer to it as "caption".